jmeter/src/main/java/org/eclipse/hono/jmeter/client/HonoReceiver.java

```diff
@@ -166,4 +171,19 @@ public void close() {
             LOGGER.error("unknown exception in closing of receiver", t);
         }
     }
+
+    private Vertx vertx() {
+        VertxOptions options = new VertxOptions()
+                .setWarningExceptionTime(1500000000)
+                .setAddressResolverOptions(new AddressResolverOptions()
+                        .setCacheNegativeTimeToLive(0) // discard failed DNS lookup results immediately
+                        .setCacheMaxTimeToLive(0) // support DNS based service resolution
```

Comment:
It's not strictly related to this PR but maybe we should avoid having these values hard coded and defining constants for them as well. Wdyt @sysexcontrol ?

Reply:
@ppatierno : I fully agree - short investigation showed that we currently have the whole code block 6 times (!) inside of Hono.
I rather would thus propose a central location (util-class e.g.)  where this code block can be called:

`        VertxOptions options = new VertxOptions()
                .setWarningExceptionTime(1500000000)
                .setAddressResolverOptions(new AddressResolverOptions()
                        .setCacheNegativeTimeToLive(0) // discard failed DNS lookup results 
immediately
                        .setCacheMaxTimeToLive(0) // support DNS based service resolution
                        .setQueryTimeout(1000));
`

And extract these values to constants additionally.
WDYT?

@balsmn : this does not affect this PR from you, of course, and will be handled separately.

Reply:
@sysexcontrol agree

Reply:
That would be good.